test/wilson/dom_test.cljs

```diff
@@ -114,12 +141,12 @@
                                    "j" {:class "warning"}
                                    "r" {:class "success"}
                                    {:class "info"}))})
-           [:table {:class "table table-hover"}
+           [:table {:class "table"}
```

Comment:
Are we sure we want table-hover to not be the default? If we change this, then the table class should be configurable; Shrieker cares that it is table-hover.